src/dotnet/APIView/APIViewWeb/Repositories/ReviewManager.cs

```diff
@@ -1,858 +1,893 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Did you intend to remove this file?

Reply:
Nope. It's not removed. It got formatted differently. Let me see If I can revert this formatting change. :(